server/scratchy_server/model/messageModel.py

```diff
@@ -13,14 +14,9 @@ class MessageModel(db_scratchy.Document):
 
 
 class MessageSchema(ma.Schema):
```

Comment:
the class need to inheritate from ModelSchema not ma.schema
and RoomModel and UserModel too